router/broker.go

```diff
@@ -117,28 +115,13 @@ func (b *broker) RemoveSession(sub *Session) {
 
 // Close stops the broker and waits message processing to stop.
 func (b *broker) Close() {
-	if b.Closed() {
-		return
-	}
 	close(b.reqChan)
```

Comment:
There are various functions sending messages over `reqChan`, such as, `Submit` and `RemoveSession` that would cause a `panic`.